### PR TITLE
[bulk]Validation: metadata and absolute link changes

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

File: docs-ref-services/latest/core-serializer-json-gson-readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
+description: Learn how to use the Azure Core Gson JSON Serializer plugin for the azure-core JsonSerializer API.
```
Comment (dereklh7) [inline docs-ref-services/latest/core-serializer-json-gson-readme.md:2]: Have title come before description in metadata.

File: docs-ref-services/latest/core-serializer-json-jackson-readme.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ---
+description: Learn how to use Azure Core Jackson JSON Serializer plugin for the azure-core JsonSerializer API.
```
Comment (dereklh7) [inline docs-ref-services/latest/core-serializer-json-jackson-readme.md:2]: Same thing here.

File: docs-ref-services/latest/cosmos-db.md [hunk 3]
```diff
@@ -1,16 +1,14 @@
 ---
+description: Learn how to Store and query key-value, JSON document, graph, and columnar data in a globally distributed database with Azure Cosmos DB.
```
Comment (dereklh7) [inline docs-ref-services/latest/cosmos-db.md:2]: Have title be first.

**Comment by v-emilypr** [comment]
**Global effort to fix build validation errors**

@alzimmermsft 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by alzimmermsft — APPROVED** [review]

**Comment by v-emilypr** [comment]
#sign-off

**Pull request closed without merging**